### DISCUSSION TITLE
Rebase option to allow `--keep-base`?

### DISCUSSION THREAD

**Discussion by Lenbok** [magit/magit#5103] [Ideas]
When interactively rebasing a branch it would be convenient to allow selecting the starting point using `--keep-base`. Perhaps a transient option, or perhaps this could automatically happen when point is on HEAD?

**Comment by tarsius** [comment]
I'll have to think about that more and currently don't have the capacity to do so.

Meanwhile you can add the argument yourself as described at https://magit.vc/manual/transient/Modifying-Existing-Transients.html.
